### DISCUSSION TITLE
Why free.fullspeed.tv does not play nice with Windows Media Player?

### DISCUSSION THREAD

**Discussion by cafealternativo** [iptv-org/community#305] [Q&A] [answered]
I was trying some links using free.fullspeed.tv using windows media player and it does not work, but they do work with VLC.

example:
http://free.fullspeed.tv/iptv-query?streaming-ip=https://www.dailymotion.com/video/x7vh8g3

I have some code that that uses the dailymotion api to get the actual stream url and then plugs into wmp successfully. What fullspeed.tv does is quite different and I am not sure how you guys make it. Are you proxying the whole thing?
Would it be possible to make this work on wmp?

**Comment by BellezaEmporium** [comment, marked as answer]
What may block for WMP is the redirection. Once the program finds out the link, it redirects the end user to the resource. It may not work on all players sadly.

Here's the function that's linked to Q-S link redirection : 

https://github.com/LaneSh4d0w/query-streamlink/blob/578a170d387d0a5135abc70d472414a8e7810270/main.py#L44-L55

**Reply by cafealternativo** [reply]
Gotcha. Appreciate it. I will look for a workaround.

Thanks.
